src/kubelet/scraper.go

```diff
@@ -148,5 +148,5 @@ func (s *Scraper) IncCurrentReruns() {
 
 // Check whether the max number of kubulet scraper reruns has been reached or not.
 func (s *Scraper) IsMaxRerunReached() bool {
-	return s.currentReruns > s.config.ScraperMaxReruns
+	return s.currentReruns > s.config.Kubelet.ScraperMaxReruns
```

Comment:
can you pull this fix out into a separate PR?

Reply:
Sure I will make a separate PR for this.

BTW, This is not a fix. It just changes the way the code refers to ScraperMaxReruns, which seems more explicit.

Reply:
pulled out